src/app/sb-admin/service/organization-list.service.ts

```diff
@@ -26,4 +26,14 @@ export class OrganizationListService {
     })
  return this.http.post(config.URLS.ORGANIZATION_CREATE_URL,body,{headers: header})
 }
+
+addSubOrg(body : any): Observable<any>{
```

Comment:
format code using vscode prettier extension

Reply:
sir I am checking this extension